warehouse/query-core/src/test/java/datawave/query/jexl/functions/ContentFunctionQueryTest.java

```diff
@@ -249,6 +250,7 @@ public void withinTestWithAlternateDate() throws Exception {
         MultivaluedMap<String,String> params = new MultivaluedMapImpl<>();
         params.putSingle(QUERY_STRING, queryStr);
         params.putSingle(QUERY_NAME, "contentQuery");
+        params.putSingle(QUERY_LOGIC_NAME, "EventQueryLogic");
```

Comment:
This implies the API is changing

Reply:
We're enforcing the API in the QueryExecutorBean which requires a logic name to define and create queries.